helm/net-exporter-chart/Chart.yaml

```diff
@@ -1,2 +1,2 @@
 name: net-exporter-chart
-version: 1.2.0
+version: 1.0.0-[[ .SHA ]]
```

Comment:
Without this, opsctl deploy would not find Helm package for net-exporter-chart on quay, and testing on CP cluster was not possible - seems we haven't been doing it lately.